dotnet/src/webdriver/Remote/RemoteSessionSettings.cs

```diff
@@ -35,7 +36,6 @@ public class RemoteSessionSettings : ICapabilities
         private const string AlwaysMatchCapabilityName = "alwaysMatch";
 
         private readonly List<string> reservedSettingNames = new List<string>() { FirstMatchCapabilityName, AlwaysMatchCapabilityName };
-        private DriverOptions mustMatchDriverOptions;
```

Comment:
Keeping it as a field is better rather than a property. `internal property` is still good, but it is an indicator for us like "please understand me and refactor".

Reply:
I can do this.

I am always nervous about having properties which are just getters/setters for fields, because it is a minefield for typos. The infinite-recursion bug from the other day, in `ActionSequence.InputDevice`, was an example of this. If an auto-property is possible with the exact same semantics (this one has `private set` so it is still safe), then I like to do those.

Reply:
Please do at least in scope of this PR. Then we will understand why `internal property` is introduced, probably it is used somewhere... I don't know, but it is another story.

Reply:
Done.

The property is not used anywhere, maybe it was in the past.